### ISSUE TITLE
Should use para-id 2000 as default fallback

### ISSUE THREAD

**Issue by NukeManDan** [substrate-developer-hub/substrate-parachain-template#47]
https://github.com/substrate-developer-hub/substrate-parachain-template/blob/9c65875fd7f59754c65c87662df0083cf56addf1/node/src/command.rs#L67-L69

**Comment by NukeManDan** [comment]
use expect instead: 
```
fn load_spec(&self, id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
		load_spec(id, self.run.parachain_id.expect("Missing para_id").into())
	}
```

**Comment by NukeManDan** [comment]
will be solved in #48
